app-web/cinny-desktop/autobuild/beyond

```diff
@@ -0,0 +1,8 @@
+abinfo "Installing application icon ..."
+for i in 32 128; do
```

Comment:
Slight complication (will ruin this for loop, unfortunately): upstream ships an icon of `128x128@2x` resolution(effectively 256x256). You might want to install that instead.